etc/kayobe/kolla/config/prometheus/rules/system.rules

```diff
@@ -8,28 +8,28 @@ groups:
     expr: ( ( node_filesystem_free_bytes - node_filesystem_avail_bytes ) / node_filesystem_free_bytes ) * 100 >= 80
     for: 5m
     labels:
-      severity: moderate
+      severity: alert
     annotations:
-      summary: "Disk on {{ $labels.instance }} is low on space"
+      summary: "Prometheus exporter at {{ $labels.instance }} reports low disk space"
       description: "{{ $labels.device }} is {{ $value }}% full."
 
   - alert: LowMemory
     expr: ( ( node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes ) / node_memory_MemTotal_bytes ) * 100 >= 80
     for: 5m
     labels:
-      severity: moderate
+      severity: alert
     annotations:
-      summary: "Memory on {{ $labels.instance }} is low"
+      summary: "Prometheus exporter at {{ $labels.instance }} reports low memory"
       description: "Memory is {{ $value }}% full."
 
   - alert: Overheating
     expr: node_hwmon_temp_celsius >= 100
     for: 5m
     labels:
-      severity: moderate
+      severity: alert
     annotations:
-      summary: "CPU {{ $labels.instance }} is overheating"
-      description: "CPU {{ $labels.chip }} is {{ $value }}% celcius."
+      summary: "Prometheus exporter at {{ $labels.instance }} reports overheating"
+      description: "Sensor {{ $labels.chip }} reports {{ $value }} degrees celcius."
```

Comment:
Can the `\n` characters be removed from the output? E.g from the slack channel:
```
Sensor 0000:00:1c_5_0000:04:00_0 reports 60 degrees celcius.\nSensor 0000:00:1c_5_0000:04:00_0 reports 61 degrees celcius.\nSensor 0000:00:1c_5_0000:04:00_0 reports 60 degrees celcius.\nSensor
```

Reply:
I think we can fix this under a different PR, it's not obvious to me what the problem is. We follow the example here for a new line:

https://prometheus.io/docs/alerting/notification_examples/

And also in the Slack API docs (https://api.slack.com/docs/message-formatting):

```
{
    "text": "This is a line of text.\nAnd this is another one."
}
```
I think the next step is to capture the post to Slack with tcpdump and see what that looks like.